src/sentry/static/sentry/app/components/search/sources/formSource.jsx

```diff
@@ -0,0 +1,96 @@
+import {withRouter} from 'react-router';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
+
+import {createFuzzySearch} from '../../../utils/createFuzzySearch';
+import FormSearchStore from '../../../stores/formSearchStore';
+import replaceRouterParams from '../../../utils/replaceRouterParams';
+
+class FormSource extends React.Component {
+  static propTypes = {
+    // search term
+    query: PropTypes.string,
+
+    // fuse.js options
+    searchOptions: PropTypes.object,
+
+    // list of form fields to search
+    searchMap: PropTypes.array,
+
+    /**
+     * Render function that passes:
+     * `isLoading` - loading state
+     * `allResults` - All results returned from all queries: [searchIndex, model, type]
+     * `results` - Results array filtered by `this.props.query`: [searchIndex, model, type]
+     */
+    children: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    searchOptions: {},
+  };
+
+  constructor(props, ...args) {
+    super(props, ...args);
+
+    this.state = {
+      fuzzy: null,
```

Comment:
fuzzy is a slightly confusing name to me. would you consider... fuzzyResults? or what you call it if you had to use a more descriptive name?

Reply:
hmm idk, it's not really results yet, it's an instance of a library